pkg/cluster/px/px.go

```diff
@@ -277,11 +279,6 @@ func (ops *pxClusterOps) Delete(c *apiv1alpha1.Cluster, opts *DeleteOptions) err
 		err := ops.runPXNodeWiper()
 		if err != nil {
 			logrus.Warnf("Failed to wipe Portworx local node state. err: %v", err)
-		} else {
-			err = ops.k8sOps.DeleteDaemonSet(pxNodeWiperDaemonSetName, pxDefaultNamespace)
```

Comment:
if we really fail to validate the daemon set, should we not have this delete?

Reply:
We are not deleting since that gives us a way to debug why it failed.  https://portworx.atlassian.net/browse/PWX-5130 is open so that we can capture logs of the failed pods using client-go and then delete the daemonset even on failure.